docs/guide/02-Introducing-Components.md

```diff
@@ -39,9 +38,9 @@ component =
 
   render state =
     HH.div_
-      [ HH.button [ HE.onClick \_ -> Just Decrement ] [ HH.text "-" ]
+      [ HH.button [ HE.onClick \_ -> Decrement ] [ HH.text "-" ]
       , HH.text (show state)
-      , HH.button [ HE.onClick \_ -> Just Increment ] [ HH.text "+" ]
+      , HH.button [ HE.onClick \_ -> Increment ] [ HH.text "+" ]
```

Comment:
This means we can have `HE.onClick' Increment` where `onClick' a = onClick \_ -> a`. Very nice!

Reply:
I was actually thinking, if we were to have an `HE.onClick'`, it would be to reintroduce the version that includes the `Maybe`, so perhaps not :wink:.

We could maybe have `HE.onClick_` for what you're talking about, but having `'` and `_` of each function might not be the friendliest API to discover. (I did it in affjax at one point, before I knew better, but a lot of those variants got removed eventually due to other changes anyway).

Reply:
Ah... Well, maybe as a project-specific Prelude then?

Reply:
I was thinking of another option when writing that comment earlier too, perhaps the event definitions that can "fail" or be "filtered" (those with the `Maybe`) could live in a separate module - they're required less often, so it would be no great hardship to have a second import for them.